spec/rails_helper.rb

```diff
@@ -15,7 +15,7 @@
   browser_options.args << "-headless"
   Capybara::Selenium::Driver.new(app, browser: :firefox, options: browser_options)
 end
-Capybara.javascript_driver = :headless_firefox
+Capybara.javascript_driver = :selenium
```

Comment:
let's revert this to headless_firefox, ir's creating issues in CI

Reply:
Damn, commited by accident